UKHomeOffice/sgmr-service
# 1.1.2 - SPCR-307 Fix menu bar highlighting wrong page when user relogs

## Description
When a user signed out and signed back in again, the menu bar indicated that they were on the same page that they were on when they signed out, however the user was on the Voyage Plans Dashboard.

For example, if the user signed out from the Pleasure Crafts page and logged in again, the user was on the Voyage Plans Dashboard but the menu bar indicated that they were still on the Pleasure Crafts page (please see the screenshot for details)

![sPCR - Sign out and Login defect](https://user-images.githubusercontent.com/56563329/157469325-078e4e65-4d6d-40a8-8c79-6417d3b8494c.jpeg)

## To Test
- Log in to the SPCR service
- Click on any navigation bar link that isn't voyage plans or sign out
- You should see that link highlighted
- Sign out and log back in
- You should return to the voyage plan page with the voyage plan link highlighted in the navigation bar

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
